app/app/(tabs)/HomePage.tsx

```diff
@@ -0,0 +1,80 @@
+import { ScrollView, SafeAreaView, View, Text, StyleSheet } from "react-native";
+import SearchBar from "../../components/SearchBar";
+import CategorySection from "../../components/CategorySection";
+import FoodSection from "../../components/FoodSection";
+import { colors, getStyles } from "../../constants/styles/global";
+
+
+const HomePage = () => {
+
+    const { theme } = getStyles();
+
+    const sampleRestaurants = [
+        {
+            "name": "The Gourmet Hut",
```

Comment:
To populate the Home page, you can utilize the Firebase database to fetch the required data.
<img width="1232" alt="Screenshot 2024-02-13 at 12 58 48 PM" src="https://github.com/SENG480a-NutriDine/mobile/assets/92160564/d5f99e4f-caf5-4d37-87d5-6f95490ee531">

Reply:
My PR was just merged - on latest main you'll see a `useRestaurants` hook you can use to handle the fetch.

For the time being, restaurants & menus need to be manually added via the firebase console, only foods can be submitted through the form

Reply:
Sounds good John! I'll work on updating the homepage such that the data is populated via the database